bitnami/metallb/templates/speaker/daemonset.yaml

```diff
@@ -167,6 +205,114 @@ spec:
             {{- if .Values.speaker.extraVolumeMounts }}
             {{- include "common.tplvalues.render" (dict "value" .Values.speaker.extraVolumeMounts "context" $) | nindent 12 }}
             {{- end }}
+        {{- if .Values.speaker.frr.enabled }}
```

Comment:
I don't see any shared volume with the `metallb-speaker` container. In the upstream project we can see the reloader volume shared with the speaker container. Could you review it?:
https://github.com/metallb/metallb/blob/b612de6055d03c155674f5adeba88149ac8462fc/charts/metallb/templates/speaker.yaml#L348-L351

Reply:
Good catch !. Thanks.